docs/pages/_document.js

```diff
@@ -39,7 +39,7 @@ export default class MyDocument extends Document {
     const { canonicalAsServer, userLanguage } = this.props;
 
     return (
-      <Html lang={userLanguage}>
+      <Html lang={userLanguage} prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article#">
```

Comment:
It doesn't seem required or related: https://stackoverflow.com/questions/27943451/meaning-of-html-prefix-attribute-open-graph-protocol/36229681#36229681.

```suggestion
      <Html lang={userLanguage}>
```